scripts/build.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const del = require('del');
+const rollup = require('rollup');
+const babel = require('rollup-plugin-babel');
+const eslint = require('rollup-plugin-eslint');
+const uglify = require('rollup-plugin-uglify');
+const pkg = require('../package.json');
+
+const bundles = [
+  {
+    format: 'umd',
+    dest: 'build/stardog.min.js',
+    plugins: [
+      eslint(),
+      babel({
+        exclude: 'node_modules/**',
+        presets: ['es2015-rollup'],
+      }),
+      uglify(),
```

Comment:
We'll also want to publish a non-minifield version. Most people (us included) just minify everything in one shot so our users will want a non-minified version as well.